queries.sql

```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name like '%mon'
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name From animals WHERE neutered=true AND (escape_attempts < 3);
+
+SELECT date_of_birth FROM animals WHERE (name='Agumon' Or name='Pikachu');
```

Comment:
- You have done great so far, but please attach a screenshot for the result to this query, according to the project requirements 🙏